### DISCUSSION TITLE
OMI-Chan Working Agreement (draft)

### DISCUSSION THREAD

**Discussion by mrmetaverse** [omigroup/omi-chan#4] [General]
## Team Norms
_Team Norms are explicit agreements on how we’ll behave together and what we’ll do if an agreement is broken. It’s everyone’s responsibility to set and uphold these agreements, but it’s truly the Team Leader’s role to ensure the team lives by these norms._

## 1. Team Ground Rules 
(incomplete, needs to be looked over and refined):

1. Mention to others when working on content related to this group
2. All assets contributed to the group must be remixable (looking for a set of CC licenses that are approved)
3. Log all assets and tools used including attribution

## 2. When we meet / Ceremonies 
This group meets weekly on Saturday at 20:00 - 21:00 UTC beginning 01/29/22
Retrospective: TBD 

## 3. Keeping Ourselves Accountable
1. Progress update every week at OMI meetings
2. We will announce goals when created to the whole of OMI
3.  We will publish avatars in progress on webpage and how-to for avatar creation/conversion

**Comment by technobaboo** [comment]
For team rules (incomplete, needs to be looked over and refined):
1. Mention to others when working on content related to this group
2. All assets contributed to the group must be remixable (looking for a set of CC licenses that are approved)
3. Log all assets and tools used including attribution

**Comment by technobaboo** [comment]
For ceremonies, meetings at 20-21 UTC (subject to change) including dedicated time to share progress of avatars.

**Reply by fire** [reply]
Clarification of every week starting on 2022-01-29?

**Reply by mrmetaverse** [reply]
@technobaboo great work again, I will admit, I am disappointed that this group did not decide to do a retro. I will attend a future meeting and explain why it matters. Its arguably the most important meeting for any working group :) Leaving it as TBD for now.

**Reply by technobaboo** [reply]
I did ask, but the thought was generally that retro didn't make sense because any independent work was hard to get out of sync :p

**Reply by technobaboo** [reply]
> Clarification of every week starting on 2022-01-29?

yea that works @fire

**Comment by technobaboo** [comment]
For keeping accountable:
1. Progress update every week at OMI meetings
2. Announce goals when created to the whole of OMI

**Reply by technobaboo** [reply]
Also publish avatars in progress on webpage and how-to for avatar creation/conversion

**Comment by mrmetaverse** [comment]
Working agreement updated! I will add to readme :rocket:

**Comment by fire** [comment]
Push for candidates for an OMI Chan team leader. Can we advertise this?

**Comment by madjin** [comment]
Wondering if M3 avatar interop group could fill the shoes of this working group: https://github.com/M3-org/avatar-interop 
Our monthly calendar includes OMI event
![brave_cChYkERVrD](https://user-images.githubusercontent.com/32600939/181834955-33d5c453-49ed-4272-8ad5-a22a15c6c705.png)
s
We have been meeting weekly for 4 months, and our template is based on OMI working group template
I setup a wiki for avatars, animations, and wearables: https://github.com/M3-org/avatar-interop/wiki/Avatars
We're down to setup workshops for fleshing out OMI-chan and friends as interoperable avatars (VRM)

**Reply by madjin** [reply]
Avatar Interop Group opening statement (in the README):
> This meetup group is dedicated to avatar interoperability R&D. We experiment with interoperable file formats, wearables, and animation systems between game engines and virtual worlds. While testing we also aim to provide useful feedback to other open metaverse standards working groups, such as the [OMI scripting group](https://github.com/omigroup/omi-scripting-group) and the [OMI glTF-extensions group](https://github.com/omigroup/gltf-extensions).
